modules/gapi/src/backends/ie/giebackend.cpp

```diff
@@ -572,9 +572,10 @@ static void setBlob(InferenceEngine::InferRequest& req,
 static void setROIBlob(InferenceEngine::InferRequest& req,
```

Comment:
BTW, why it's handled only for `ROI`? I believe our main use case when:
* IE device - `GPU`
* User passed remote context
* MediaFrame comes with `VPL` adapter

In that case we have to do preprocessing on our end since `IE` isn't able to do it for us...

Reply:
In not-long time ago there was simple function `SetBlob` in a way `SetBlob(...make_shared_blob(Blob, roi)...)` which actually prepare ROI Blob from it's argument - which doesn't work in case of GPU blob
Now `SetROIBlob` take care about decision: prepare ROI blob (legacy way) or not (GPU approach)